mangaki/mangaki/settings.py

```diff
@@ -253,6 +251,10 @@ def get_sentry_handler(config_instance) -> str:
 
 STATIC_ROOT = config.get('deployment', 'STATIC_ROOT', fallback=os.path.join(BASE_DIR, 'static'))
 MEDIA_ROOT = config.get('deployment', 'MEDIA_ROOT', fallback=os.path.join(BASE_DIR, 'media'))
+DATA_ROOT = config.get('deployment', 'DATA_ROOT', fallback=os.path.join(BASE_DIR, 'data'))
+
+DATA_DIR = DATA_ROOT
```

Comment:
FIXME: replace every occurrences of `DATA_DIR` by `DATA_ROOT`

Reply:
Plz noes je vais forcément en oublier

Reply:
Juste ajoute un FIXME j'ai dit

Reply:
Done.